monitor/go.mod

```diff
@@ -0,0 +1,28 @@
+module github.com/gofiber/contrib/monitor
+
+go 1.22.6
+
```

Comment:
Should be 1.22 without the patch

Reply:
f97a389

Reply:
[1001bc8](https://github.com/gofiber/contrib/pull/1172/commits/1001bc8815e4b71b903bc0ddb2a78597efe1d648)